build-system/tasks/extension-generator/bento/amp-__component_name_hyphenated__/__component_version__/amp-__component_name_hyphenated__.css

```diff
@@ -0,0 +1,19 @@
+/**
```

Comment:
Should we maybe generate JSS here instead? Included also as `shadowCss` on the AMP definition.

Reply:
Both CSS and JSS files are generated, based on the screenshot in this issue: https://github.com/ampproject/amphtml/issues/31887 (which comes from the "Creating a Bento extension" docs)
Will Add the `shadowCss` to the AMP definition.

Reply:
Addressed in https://github.com/ampproject/amphtml/pull/31965